ngraph/frontend/onnx/frontend/src/op/qlinear_conv.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// Disabled in CMakeList
+// Update to higher opset required
+
+#include "op/qlinear_conv.hpp"
+
+#include <cstddef>
+#include <memory>
+#include <vector>
+
+#include "conv.hpp"
+#include "dequantize_linear.hpp"
+#include "exceptions.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "onnx_import/core/null_node.hpp"
+#include "quantize_linear.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector qlinear_conv(const Node& node) {
+    const OutputVector& inputs = node.get_ng_inputs();
+
+    auto x = inputs[0];
```

Comment:
You should check the number of inputs before accessing them the way you do below. This might segfault.